src/common-interfaces.ts

```diff
@@ -0,0 +1,24 @@
+// TODO figure out interface for node
+export interface RSNode {
+  [key: string]: any;
+}
+
+export interface RSNodes {
+  [key: string]: RSNode;
+}
+
+export interface RSEvent {
+  [key: string]: string;
+}
+
+export interface ChangeObj {
+  [key: string]: any;
+}
+
+export interface QueuedRequestResponse {
+  statusCode: number;
+  body?: object;
+  contentType?: ContentType
+}
+
+export type ContentType = any;
```

Comment:
Shouldn't that always be a string?

Reply:
That's meant to be a placeholder until I figure out what ContentTypes we're expecting.